src/pages/Cabinet/components/CabinetModal.tsx

```diff
@@ -80,6 +96,16 @@ export const CabinetModal: FC<CabinetModalProps> = ({ open, onUpdate, onClose })
     },
   ];
 
+  useEffect(() => {
+    if (error) {
+      setResponseError((error as any).message);
+    }
+    if (data) {
```

Comment:
Dont  check  data  in  this  case  ..  You  can use  isSuccess  flag  const [createCabinet, { data, error, isSuccess }] = useCreateCabinetMutation();